src/test/resources/org/jenkinsci/plugins/vsphere/tools/expected_output.yml

```diff
@@ -13,7 +13,7 @@
       labelString: "windows vsphere"
       launchDelay: 60
       launcher:
-        jnlp:
+        inbound:
```

Comment:
Adapting to a Jenkins core change that renamed this to "inbound" some time back.